docs/zkapps/lightnet.mdx

```diff
@@ -0,0 +1,218 @@
+
+- `/root/logs/*.log`
+- `/root/.mina-network/mina-local-network-2-1-1/nodes/**/logs/*.log`
+
+To save the log files that are produced by Docker container processes to the host machine file system:
+
+```sh
+zk lightnet logs save
+```
+
+To debug your zkApp, you can follow the Docker container process logs in real time. 
+
+On a different terminal window, to stream the process logs:
+
+```sh
+zk lightnet logs follow
+```
+
+Select the Docker container process to follow. Press Ctrl+C to stop streaming.
+
+
+## Lightnet status
+
+To get the network status:
+
+```sh
+zk lightnet status
+```
+
+The network status is returned, including useful URLs. For example, the default single node network status returns these useful URLs:
+
+```text
+- Mina Daemon GraphQL endpoint │ http://localhost:8080/graphql
+- Accounts Manager endpoint    │ http://localhost:8181
+- Archive-Node-API endpoint    │ http://localhost:8282
+- PostgreSQL connection string │ postgresql://postgres:postgres@localhost:5432/archive
+```
+
+Use these URLs when you configure your zkApp.
+
+Lightnet status includes blockchain network properties, the Docker container state, and a code snippet of a zkApp using o1js API. 
+
+Blockchain network properties include:
+
+- Sync status
+- Commit ID
+- Chain ID
+- Consensus mechanism
+- Consensus configuration
+  - Transaction finality ("k" blocks)
+  - Slot duration (new block every ~)
+  - Slots per Epoch
+- SNARK work fee
+- Known accounts
+- Uptime
+
+## Accounts
+
+Each Docker image is packaged with a genesis ledger that is configured with more than 1000 prefunded accounts. Each account has a balance of 1550 MINA.
+
+The Mina accounts manager helper tool provides a random public/private key pair that operates with accounts already configured in the genesis ledger. 
+
+**Advanced** To access the genesis ledger configuration file `daemon.json`, you must manually access the Docker container file system with `docker exec -it mina-local-lightnet bash`. 
+
+### zkApp Account Interactions
+
+The `acquireKeyPair()`, `releaseKeyPair()`,  and `listAcquiredKeyPairs()` methods in the `Lightnet` o1js API namespace handle zkApp access to account information, including public keys, private keys, and secret keys. For details, see the [doc comments](https://github.com/o1-labs/o1js/blob/f5db874a3fd52635d3d77628696bc8e97f78b76c/src/lib/fetch.ts#L1012) in the code. 
+
+For a real-world example of using Lightnet accounts, see [run_live.ts](https://github.com/o1-labs/o1js/blob/main/src/examples/zkapps/hello_world/run_live.ts) example file used in [Tutorial 1: Hello World](/zkapps/tutorials/hello-world).
+
+### Account Use Other Than zkApp
+
+To get the accounts information outside of the zkApp program, use the Accounts-Manager Docker container service at the endpoint: http://localhost:8181/. This endpoint is the same for all users and is available when the Lightnet Docker container is up and running.
+
+<!-- 
```

Comment:
@Sventimir  how do we release the unused accounts?

Reply:
I will benefit from your expertise and pairing to work through the accounts

Reply:
If we are talking not about the o1js API then you can refer to https://github.com/shimkiv/mina-accounts-manager readme where available HTTP endpoints presented.
`GET http://localhost:8181/acquire-account`
`PUT http://localhost:8181/release-account (with json body)`